e2e-test/e2e.py

```diff
@@ -97,6 +102,166 @@ def test_post_resource_increase_count(
+                actual_entity_resource_ref = _getDict(actual_compartment_value).get("reference")
+
+                expected_compartment_value = (expected_entity_resource_ref.split("/")[0] 
+                                    if isinstance(expected_entity_resource_ref, str) else "")
+                actual_compartment_value = (actual_entity_resource_ref.split("/")[0] 
+                                    if isinstance(actual_entity_resource_ref, str) else "" )
+
+                if expected_compartment_value != actual_compartment_value:
+                    raise AssertionError(
+                        "Field '{}' compartment owner mismatch in AuditEvent:\n"
+                        "Expected: {}\n"
+                        "Actual: {}".format(
+                                            field, expected_entity_resource_ref, 
+                                            actual_entity_resource_ref
+                                            )
+                    )
+
+                #check if the AuditEvent is for the correct resource type
+                expected_entity_what = (_getDict(expected_value[2])["what"] 
+                if isinstance(expected_value, list) and len(expected_value) > 2 else "")
+                actual_entity_what = (_getDict(actual_value[2])["what"] 
+                if isinstance(actual_value, list) and len(actual_value) > 2 else "")
+                
+                expected_entity_resource_ref = _getDict(expected_entity_what).get("reference")
+                actual_entity_resource_ref = _getDict(actual_entity_what).get("reference")
+
+                expected_value = (expected_entity_resource_ref.split("/")[0] 
+                                    if isinstance(expected_entity_resource_ref, str) else "")
+                actual_value = (actual_entity_resource_ref.split("/")[0] 
+                                    if isinstance(actual_entity_resource_ref, str) else "" )
+
+            elif field == "recorded":
+                expected_value = date.today()
+                actual_value = (datetime.fromisoformat(actual_value)
+                                 if isinstance(actual_value,str) else datetime.min).date()
+            elif field == "period":
+                #confirm with period.end that object is correct
+                actual_period_end = _getDict(actual_value).get("end")
+                expected_value = date.today()
+                actual_value = (datetime.fromisoformat(actual_period_end)
+                                if isinstance(actual_period_end,str) else datetime.min).date()
+            
+            if actual_value != expected_value:
+                raise AssertionError(
+                    "Field '{}' mismatch in AuditEvent:\n"
+                    "Expected: {}\n"
+                    "Actual: {}".format(field, expected_value, actual_value)
+                )
+    
+    logging.info("All AuditEvent fields verified successfully")
+
+def _getDict(raw: Any) -> Dict[str, str]:
```

Comment:
nit: (please see the [style guide naming conventions](https://google.github.io/styleguide/pyguide.html#316-naming))
```suggestion
def _get_dict(raw: Any) -> Dict[str, str]:
```